guides/v2.2/frontend-dev-guide/css-topics/css-overview.md

```diff
@@ -32,7 +32,7 @@ There are a couple options to help with CSS and site performance.
 
 * Minification of CSS files reduces the file size being sent. It does this by stripping white space within the file.
 
-To enable / disable these settings, go into Admin > **Stores** > Setting > **Configuration** > **Advanced** > **Developer** > **CSS Settings**.
+To enable / disable these settings, go into **Admin** > **Stores** > **Settings** > **Configuration** > **Advanced** > **Developer** > **CSS Settings**.
```

Comment:
Since those two words that you bolded are technically not something a user would click, I suggest cleaning this entire sentence:

```suggestion
You can enable or disable these settings in the _Admin_ panel.  Click **Stores** and, in the _ Settings_ section, click **Configuration** > **Advanced** > **Developer** > **CSS Settings**.
```

Reply:
Telling the developer "and in the _Settings_ section", seems too wordy, and can rather be communicated in this better way, perhaps?
To enable or disable these settings, go to _Admin_ panel, click **Stores** > **Settings::Configuration** > **Advanced** > **Developer** > **CSS Settings**.

@rogyar Any input on this from your side? Thanks.

Reply:
Sure! But definitely remove that forward slash. 🙂